### PR TITLE
Adds the Contender G13

### PR DESCRIPTION
**Pull request by ArcLumin**
Adds in the Contender G13! It can only hold one bullet at a time, but can use any type of ammo

imageimage

It's added to the bartender's starting items, as a buff, and a backup if his shotty is stolen.

It's also purchasable by traitors for 10 TC with a fun skin (10 TC because they can use all sorts of ammo), and in the russian room on the derelict.

The gun is based off of this; https://en.wikipedia.org/wiki/Thompson/Center_Contender

### CODE REVIEW

**Comment by MethMedic** [comment]
Love it

**Comment by Spacedong** [comment]
conflix

**Comment by Spacedong** [comment]
very cool idea, doesn't seem that overpowered. but, can it fire grenades?

**Comment by ArcLumin** [comment]
Haven't tested it yet as I'm traveling and using my phone's connection to check shit. I can't download the code yet because of the data caps.

Also @spacedong if you get the grenade launcher ammo, it should

**Comment by Spacedong** [comment]
>desc = "The Contender G13, a favorite amongst space hunters. An easily modified bluespace barrel and break action loading means it can use any ammo available! It can only hold one bullet at a time, however"
there is no period at the end of this sentence

**Comment by MethMedic** [comment]
No period? Close it up boys don't merge

**Comment by KayeArray** [comment]
get em spacedongo

**Comment by ArcLumin** [comment]
I'll just not put a period there to piss off dongo

**Comment by KayeArray** [comment]
do you have to pump this?

**Comment by ArcLumin** [comment]
Uh.

No? I didn't even think about that. I originally didn't have it under shotgun child. Let's see

**Comment by ArcLumin** [comment]
yeaaaaah didn't think about that when I made it a child of the shotgun. Lemme fix that

**Comment by Spacedong** [comment]
DNM until the period is placed on the end of that description

**Comment by MethMedic** [comment]
Double barrel not double action wew

**Comment by Spacedong** [comment]
like this https://vgy.me/yax9sE.jpg @MethMedic ?

**Comment by ArcLumin** [comment]
https://en.wikipedia.org/wiki/Break-action

**Comment by ArcLumin** [comment]
There, it no longer needs to be pumped

**Comment by ArcLumin** [comment]
I'll need to test this out when I actually get home and see if all ammo works for it. I won't be back till tomorrow though. Can someone else test it out?

**Comment by KittyKathy** [comment]
I don't think you should give this to nuke ops or remove the pAI Card in the nuke ops shuttle.
The pAI Card is useful for when you bomb the AI but it is still alive and you want to take it out quickly/save ammo before it can call for help.

The reason why this should not be available to nuke ops is because I don't think this comes with a syndicate pin nuke ops should get a version of it with the syndicate pin. If you let them have this gun they can buy sniper .50 ammo that I think deletes/destroys stuff shot by the .50 ammo and just take the gun and break into Blastco like how you used to be able to do with sniper rifles, until all syndicate guns came with a syndicate firing pin so you can't shoot them on the syndicate base. 

It would take like 10 tc which is two ammo magazines of .50 sniper ammo to break into Blastco with this gun or maybe 5 tc if you are daring enough to...
1. Stand next to the First Blastco door alt click the shutters so you can click the floor from the list it brings up with a empty bag in hand.
2. Activate a minibomb and quickly put it in then empty bag then drag the bag onto the floor from the list so it empties the contents onto the floor which should make it under the shutters.
3. Run from the bomb which should destroy the first shutters and cause a breach but you were wearing a space suit right? The explosion does not touch any bomb cores so everything is okay, now finish off the second door with the sniper .50 ammo and you're in Blastco.

The origin tech levels( origin_tech = "combat=6;materials=4;bluespace=5" ) might be too good since nothing on station I think had combat 6. You always needed to order combat shotguns for combat 6 or get syndicate/antag gear but since it is a unique item( 1 in a normal round, 2 in a nuke ops round) it should be fine. 

Other than the possible issues I see with it, the gun is pretty cool and should be added.

**Comment by oprayx77** [comment]
yeah this shouldnt be a nuke op weapon

they have too much variety with the ammo they have and can abuse it really well
idea: give it to security? put it in a glass box in the hos' office?

**Comment by ArcLumin** [comment]
The original idea was to put it on the derelict, but I didn't want conflicts. As for the nuke ops break in, the bomb payloads in there are designed to explode if it detects movement when the hatch door isn't properly activated (or, its supposed to). The additional tech levels are leverage for the bartender vs. sci, and allows for an interesting dynamic.

**Comment by ArcLumin** [comment]
But yeah, nuke op weapon was an afterthought, I thought itd be a neat tator weapon. I'll rework this shit tomorrow.

**Comment by KittyKathy** [comment]
The bomb payloads only explode when something triggers it like a bomb payload from a syndicate bomb, only bombs /shooting it maybe will trigger it.
If you are going to give it to nuke ops make it a child of the one the bartender gets and add a syndicate firing pin to it so it can't be used on the base like the rest of guns they start with(not counting the ion rifle).

**Comment by ArcLumin** [comment]
Ah, I was under the understanding the payloads were motion activated. And yeah, my fix was gonna be a reskinned version, with the pin. For fun, ya know.

**Comment by Anderbergst** [comment]
this is really good

**Comment by Swagile** [comment]
gunna wait for a code check and for the issues above to be fixed before i support this

**Comment by Pyko1** [comment]
Kittykathy have you actually tested that, as the bombs should activate if any of the shutters are blown up.

File: code/modules/projectiles/guns/projectile/shotgun.dm [hunk 1]
```diff
@@ -524,3 +524,18 @@
 	unique_rename = 0
 	unique_reskin = 0
 	can_sawn = FALSE
+
+/obj/item/weapon/gun/projectile/revolver/doublebarrel/contender
+	desc = "The Contender G13, a favorite amongst space hunters. An easily modified bluespace barrel and break action loading means it can use any ammo available! It can only hold one bullet at a time, however."
```
Comment (JohnGinnane) [inline code/modules/projectiles/guns/projectile/shotgun.dm]: >bullet
>can take shotgun shells and grenades

Comment (oprayx77) [reply code/modules/projectiles/guns/projectile/shotgun.dm]: name it cartridge 🥇

Comment (ArcLumin) [reply code/modules/projectiles/guns/projectile/shotgun.dm]: Or round.

**Comment by JohnGinnane** [comment]
As mentioned this will probably require you to pump the gun when you load it due to shotgun behaviour, is this intended?

**Comment by ArcLumin** [comment]
If I recall, this is not the case anymore, but I'll need to test when I return.

**Comment by ArcLumin** [comment]
Lazy, syndie version of it; https://puu.sh/t8gnK.png

**Comment by ArcLumin** [comment]
Okay, fixed a bunch of shit in it, now it's good to go. Added it to the derelict, guarded by a russian. New reskinned version in nuke ops with the proper firing pin. Changed the word bullets to rounds

**Comment by Hackole** [comment]
I don't like this for nukeops. This means they could have a grenadelauncher without having to actually buy the strider, which is pretty immense powercreep

**Comment by ArcLumin** [comment]
Now what will I do with this slightly recolored sprite rip

**Comment by Hackole** [comment]
Reskin option?

**Comment by SpankMasterson** [comment]
I worry if it's too OP, arc is reviewing this and once we know how it functions better I am willing to +1

**Comment by ArcLumin** [comment]
If it's too op I'll remove it from nuke ops. As for bartender, well, bartender needs a few buffs.

**Comment by ArcLumin** [comment]
Added the contender to the summon guns event. Also fixed a spawning typo

**Comment by KittyKathy** [comment]
Actually they can just use the gun from the 15tc bundle with the under barrel grenade launcher if they wanted to blow stuff up without a strider it even comes with a ammo box of grenades to reload.
Not really that op compared to that bundle which is only 5 tc more.

**Comment by Kierany9** [comment]
Bundles were a mistake

**Comment by Swagile** [comment]
medical bundle is not a mistake, fk u

**Comment by Hackole** [comment]
Acceptable, +1

**Comment by ghost** [comment]
Sure why not? +1

**Comment by ArcLumin** [comment]
GIMME MUH CODE REVIEW

**Comment by TDcoolguy300** [comment]
Code looks fine, nothing is popping out, you tested this, right?

**Comment by ArcLumin** [comment]
That I did. Works like a charm

**Comment by ArcLumin** [comment]
Considering TD gave his code review, and we have head okays, merging

**Pull request merged**